nlfyappapi/routes/announcements.js

```diff
@@ -8,6 +8,16 @@ module.exports = router;
 const Model = require("../model/announcementModel");
 const NotificationModel = require("../model/notificationsModel");
 
+router.use(function (req, res, next) {
+  res.header("Access-Control-Allow-Origin", "http://localhost:3001");
```

Comment:
Instead of hardcoding the URI as "http://localhost:3001", can you define this in .env file and read the variable from there in all the files where this is used? 
This will be easier for us when we deploy this code in the AWS Server.We only have to change in our .env.

Reply:
Added the .env file.